Assets/Scripts/Interaction Scripts/ControllerState.cs

```diff
@@ -121,9 +168,34 @@ public class ControllerState : MonoBehaviour {
 			return LeftThumbAngle;
 		}
 
-		public Transform GetLeftOrigin() /// The origin ("position") of the left controller.
+		public bool GetLeftThumbMoved() /// True if LeftThumbDelta{.x|.y} != 0.0f or LeftThumbAngle != 0.0f, meaning that the left thumbstick has been moved.
```

Comment:
Same more importantly for the public functions that will be called, (< summary > tag)